src/RegExpr.pas

```diff
@@ -1575,9 +1574,9 @@ class function TRegExpr.InvertCaseFunction(const Ch: REChar): REChar;
 
 procedure TRegExpr.SetExpression(const AStr: RegExprString);
 begin
-  if (AStr <> fExpression) or not fExprIsCompiled then
+  if (AStr <> fExpression) or not regexpCompiled then
```

Comment:
in fact this is very strange looking - past form instead of passive

Reply:
renamed to regexpIsCompiled, ok.